samples-lib/src/main/java/com/esri/arcgismaps/sample/sampleslib/DownloaderActivity.kt

```diff
@@ -224,59 +231,56 @@ abstract class DownloaderActivity : AppCompatActivity() {
                 emit(LoadStatus.FailedToLoad(it))
                 return@flow
             }
+            // set up the file at the download path
+            val destinationFilePath = File(provisionLocation.path, portalItem.name).toString()
```

Comment:
Since you require a type `File` further down in your code, you might as well hold on to a `File` instance instead of `String`:
```suggestion
            val destinationFilePath = File(provisionLocation.path, portalItem.name)
```